config/main.py

```diff
@@ -788,6 +798,70 @@ def del_vlan_member(ctx, vid, interface_name):
     db.set_entry('VLAN', vlan_name, vlan)
     db.set_entry('VLAN_MEMBER', (vlan_name, interface_name), None)
 
+@vlan.group('dhcp_servers')
+@click.pass_context
+def vlan_dhcp_servers(ctx):
+    pass
+
+@vlan_dhcp_servers.command('add')
+@click.argument('vid', metavar='<vid>', required=True, type=int)
+@click.argument('dhcp_server_ip', metavar='<dhcp_server_ip>', required=True, type=str)
+@click.pass_context
+def add_vlan_dhcp_servers(ctx, vid, dhcp_server_ip):
+    """ Add a DHCP server IP helper under the vlan """
+    if not is_ipaddress(dhcp_server_ip):
+        click.echo('Invalid ip address')
+        return
+    db = ctx.obj['db']
+    vlan_name = 'Vlan{}'.format(vid)
+    vlan = db.get_entry('VLAN', vlan_name)
+
+    if len(vlan) == 0:
+        click.echo("{} doesn't exist".format(vlan_name))
+        raise click.Abort()
+    dhcp_servers = vlan.get('dhcp_servers', [])
+    dhcp_server_ip = str(dhcp_server_ip)
+    if dhcp_server_ip in dhcp_servers:
+        click.echo("{} is already a member of {}".format(dhcp_server_ip,vlan_name))
+        raise click.Abort()
+    else: 
+        click.echo("{} is being added as a dhcp address to {}".format(dhcp_server_ip,vlan_name))
```

Comment:
Make message less verbose and print message after the operation, e.g., `Added DHCP server address {} to VLAN {}` or `Added DHCP IP helper address {} to VLAN {}`. We also need to unify the language here. 

Also please add space after comma in `format(dhcp_server_ip, vlan_name)`